src/notification/notification.service.ts

```diff
@@ -0,0 +1,161 @@
+import { Injectable, Logger } from '@nestjs/common';
+import * as cron from 'node-cron';
+import { ConfigService } from '@nestjs/config';
+import { DiscordService } from 'src/discord/discord.service';
+import { SubscriptionService } from 'src/subscription/subscription.service';
+import { Subscriber, Subscription } from 'prisma/prisma.types';
+
+@Injectable()
+export class NotificationService {
```

Comment:
Could put this into the subscription module as `subscription.notifier.ts` potentially.

Reply:
I think that's a good call. These are anyway tightly coupled, so I guess no need for separate modules.

Reply:
Actually, this causes a circular dependency, so instead, I just renamed the module to SubscriptionNotifier to emphasise that they are close. A bit long though, what do you think?

Reply:
Fine for now. I think we could look more into the circular dependency but should be good for now.